src/Microsoft.IdentityModel.JsonWebTokens/JwtTokenUtilities.cs

```diff
@@ -278,6 +284,87 @@ internal static object GetClaimValueUsingValueType(Claim claim)
             return claim.Value;
         }
 
+        /// <summary>
+        /// Gets ValueType of the claim from it's Value.
+        /// </summary>
+        /// <param name="value">Represents value of a claim.</param>
+        /// <returns>String representing claim's ValueType.</returns>
+        internal static string GetClaimValueTypeFromValue(object value)
```

Comment:
Added this method in JwtTokenUtilities to allow token specific ClaimTypes matching.

Reply:
I think the comment here sounds a little confusing. Could it just be "Gets the value type of the claim"?

I think the comment for the param would be a little simpler if it was just:
`<param name="value">The claim value.</param>`

And same for the return value:
`<returns>The value type of the claim.</returns> `

Reply:
I agree. Sounds simple and easy to understand.